### ISSUE TITLE
about 'fsrcnn'

### ISSUE THREAD

**Issue by T5eng** [IvoryCandy/super-resolution#19]
optimizer should be SGD instead of Adam
learning rate of fsrcnn should be 0.001
weight init is not proper.
Hyper parameter in Transposed Conv layer is not correct, causing error when changing upscale factor.

**Comment by fangangping** [comment]
have you fixed this problem ? I find fsrcnn can not work when i set the upscale factor to 8.

**Comment by T5eng** [comment]
I think we need to adjust the "outpadding" argument according to different upscale factor. you can check out the pytorch document about ConvTranspose2d.
